client/my-sites/posts/post.jsx

```diff
@@ -381,4 +380,4 @@ export default connect(
 		};
 	},
 	{ setPreviewUrl, setLayoutFocus }
-)( localize( Post ) );
+)( updatePostStatus( localize( Post ) ) );
```

Comment:
Might it be worth taking the opportunity to compose these HoCs in this PR or tackle that later?